Library/Homebrew/formula_installer.rb

```diff
@@ -465,15 +465,21 @@ def expand_requirements
     [unsatisfied_reqs, deps]
   end
 
+  def dependencies_to_pour_bottles
+    []
+  end
+
   def expand_dependencies(deps)
     inherited_options = Hash.new { |hash, key| hash[key] = Options.new }
+    poured_bottle = pour_bottle?
 
     expanded_deps = Dependency.expand(formula, deps) do |dependent, dep|
       inherited_options[dep.name] |= inherited_options_for(dep)
       build = effective_build_options_for(
         dependent,
         inherited_options.fetch(dependent.name, []),
       )
+      poured_bottle = true if install_bottle_for?(dep.to_formula, build)
```

Comment:
This isn't a safe assumption; the bottle pouring may have failed.

Reply:
We haven't yet poured any bottles. We're testing at this point whether we plan to pour any bottles. This variable was named poorly. I've renamed it to `pour_bottle`.